### PR TITLE
BD-1029:Rework Image Licensing in Lightning

### PR DESCRIPTION
**Pull request by BhargaviPalineni79**
Requested Changes have been updated.

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -8,5 +8,14 @@ within the component is also subject to the terms and conditions
 of these licenses. The LICENSE file contains the text of all the
 licenses which apply within this component.
 
-Some images come from https://www.pexels.com and are licensed under the Pexels License at: https://www.pexels.com/license/
-Credits: Scott Webb, Pixabay 
+The unsplash-images directory within this component contains images which are licensed under the Pexels license.  The unsplash-images
```
Comment (mhughesacn) [inline NOTICE:11]: Swap these around: unsplash images and unsplash license, not Pexels.

File: NOTICE [hunk 2]
```diff
@@ -8,5 +8,14 @@ within the component is also subject to the terms and conditions
...
+directory contains its own license file, called LICENSE, which describes the licensing of the
+material in the unsplash-images directory.
+
+These images derived from pixabay on pexels.com are licensed under the CC0 1.0 Universal License
+https://www.pexels.com/photo/architecture-buildings-business-city-290595/
+https://www.pexels.com/photo/time-lapse-photography-of-waterfalls-during-sunset-210186/
+
+The pexels-images directory within this component contains images which are licensed under the Unsplash license.  The pexels-images
```
Comment (mhughesacn) [inline NOTICE:19]: Swap: Pexels and Unsplash license -> Pexels and Pexels license.

File: LICENSE [hunk 3]
```diff
@@ -199,3 +199,78 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
```
Comment (mhughesacn) [inline LICENSE:202]: Please add a header:

CC0 1.0 Universal License

**Review by mhughesacn — CHANGES_REQUESTED** [review]
Hi Bhargavi - some comments below.  Also, the soft links are not correct.  I think the examples/shaders/water-waves/stones.jpg link should be to ../../../pexels-images/stones.jpg and the sample image to ../../unsplash-images/sampleImage.jpg sampleImage.jpg

File: LICENSE [hunk 4]
```diff
@@ -199,3 +199,86 @@
...
+***************************************************************************************************
```
Comment (mhughesacn) [inline LICENSE:203]: I just want the license to have a title: so please remove 203-212 and add a single line with the name of the license: "CC0 1.0 Universal License" and a line of ==== to make it stand out for someone looking in the LICENSE file.

**Review by mhughesacn — DISMISSED** [review]

**Comment by mhughesacn** [comment]
Thanks Bhargavi.  You must sign the CLA so all checks pass.  Then we can ask Metrological to approve this change, or suggest directory name changes etc.

**Comment by BhargaviPalineni79** [comment]
I actually signed the CLA , But not sure why it still showing CLA not signed.

**Comment by mhughesacn** [comment]
Thanks Bhargavi.
Note for Metrological: Some images in Lightning were found to be licensed under licenses which do not allow you to relicense to another license.  Comcast Legal allow this to be handled by including these files in separate directories with the licensing indicated clearly in a particular way.  We have done this, but it requires that the original images be removed from where they were located.  The two images have been replaced by two soft links.  We see approval for this change, or an indication of preferred directory names (instead of pexels-images/unsplash-images).

File: NOTICE [hunk 5]
```diff
@@ -8,5 +8,14 @@ within the component is also subject to the terms and conditions
 of these licenses. The LICENSE file contains the text of all the
 licenses which apply within this component.
```
Comment (mhughesacn) [inline NOTICE:9]: This line is missing a section: "The LICENSE file contains the text of all the licenses which apply
within this component, except as indicated in the following paragraphs."

File: NOTICE [hunk 6]
```diff
@@ -8,5 +8,14 @@ within the component is also subject to the terms and conditions
...
 
-Some images come from https://www.pexels.com and are licensed under the Pexels License at: https://www.pexels.com/license/
-Credits: Scott Webb, Pixabay 
+The unsplash-images directory within this component contains images which are licensed under the Unsplash license.  The unsplash-images
```
Comment (mhughesacn) [inline NOTICE:11]: Need to fix the name here to examples/unsplash-images

File: NOTICE [hunk 7]
```diff
@@ -8,5 +8,14 @@ within the component is also subject to the terms and conditions
...
+directory contains its own license file, called LICENSE, which describes the licensing of the
+material in the unsplash-images directory.
+
+These images derived from pixabay on pexels.com are licensed under the CC0 1.0 Universal License
+https://www.pexels.com/photo/architecture-buildings-business-city-290595/
+https://www.pexels.com/photo/time-lapse-photography-of-waterfalls-during-sunset-210186/
+
+The pexels-images directory within this component contains images which are licensed under the pexels license.  The pexels-images
```
Comment (mhughesacn) [inline NOTICE:19]: Update the name to examples/pexels-images.

**Review by mhughesacn — CHANGES_REQUESTED** [review]
Hi Bhargavi:
The NOTICE file needs to be updated for the new paths.  The links need to be removed.  Also, the new image locations need to be included in the configuration as Wouter suggested.  I am seeking clarification on this and will report back.
Thanks.

File: NOTICE (hunk 5 above)
Comment (BhargaviPalineni79) [reply NOTICE]: Hi mhughesacn ,
Updated the NOTICE & removed the soft links for the images. But for the configuration of new images could you please suggest where do i need to include. 
Thanks

**Review by mhughesacn — APPROVED** [review]

**Comment by mhughesacn** [comment]
I am happy with these changes now.  Metrological need to approve Bhargavi's changes to config files giving new image locations.

**Pull request merged**